examples/k8s/kustomization.yml

```diff
@@ -0,0 +1,15 @@
+resources:
+- bytesafe-ce.yml
+
+namespace: bytesafe
+
+configMapGenerator:
+- name: bytesafe-config
+  literals:
+   - SMTP_USER=""
+   - SMTP_HOST=""
+   - SMTP_PORT=""
+   - REDIS_HOST=redis
+   - DB_HOST=postgres
+   - URL_PREFIX="http://localhost:8081"
+   - TELEMETRY="false"
```

Comment:
TELEMETRY  needs to be default not-false, change to ""

Reply:
Also, make sure to include all options from the ones you deleted above. (SMTP_FROM_ADDRESS etc.)